tests/test_utils.py

```diff
@@ -11,76 +11,97 @@ def test_is_url():
 
 def test_is_file():
     assert not utils.is_file(None)
-    assert utils.is_file(resource_file('dummy.nc'))
-    long_str = "".join('a' for i in range(260))
+    assert utils.is_file(resource_file("dummy.nc"))
+    long_str = "".join("a" for i in range(260))
     assert not utils.is_file(long_str)
-    assert utils.is_file(Path(resource_file('dummy.nc')))
+    assert utils.is_file(Path(resource_file("dummy.nc")))
 
 
 def test_sanitize():
-    assert utils.sanitize('output') == 'output'
-    assert utils.sanitize('My Output 1') == 'my_output_1'
-    assert utils.sanitize('a.b') == 'a_b'
-    assert utils.sanitize('a-b') == 'a_b'
-    assert utils.sanitize('return') == 'return_'
-    assert utils.sanitize('Finally') == 'finally_'
+    assert utils.sanitize("output") == "output"
+    assert utils.sanitize("My Output 1") == "my_output_1"
+    assert utils.sanitize("a.b") == "a_b"
+    assert utils.sanitize("a-b") == "a_b"
+    assert utils.sanitize("return") == "return_"
+    assert utils.sanitize("Finally") == "finally_"
 
 
 def test_delist():
-    assert utils.delist(['one', 'two']) == ['one', 'two']
-    assert utils.delist(['one', ]) == 'one'
-    assert utils.delist('one') == 'one'
+    assert utils.delist(["one", "two"]) == ["one", "two"]
+    assert (
+        utils.delist(
+            [
+                "one",
```

Comment:
```suggestion
                "one"
```
Then re-run `pre-commit run --all-files`